app/components/roadmap-page/-utils/engines.yaml

```diff
@@ -0,0 +1,25 @@
+- name: Engines
+  summary: |
+    An advanced enterprise feature that allows an application to be broken up into multiple pieces that are loosely coupled,
+    with each having a separate container to avoid naming collisions between teams of developers.
```

Comment:
Is this taken from an official RFC or the ember engines site? 

//Cc @acorncom

Reply:
Neither. I paraphrased for brevity.